### PR TITLE
Issue #277: Added call out to downloading booster _before_ trying to …

### LINKED ISSUE #277: "Deploy Using the oc CLI Client" sections assume a local clone of the booster
The "Deploy Using the oc CLI Client" sections inside "Build and Deploy Your Booster" chapters implicitly assume that the user has the booster project locally -- either cloned from GitHub or maybe unzipped from launcher. This should be made explicit. If there's a section in the documentation that details this, a link to it will be sufficient.

The only exception are the Secured boosters, where the "Build and Deploy Your Booster" have a different structure and the documentation tells the user to clone the booster directly from GitHub.

### PR DESCRIPTION
**Pull request by rhoads-zach**
…deploy it via cli. Fixes #277 

@Ladicek @inoxx03 mind taking a look?

### CODE REVIEW

File: docs/topics/oc-client-deploy-booster-download-note.adoc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[IMPORTANT]
+====
+You must still create a booster using {launcher} before you can deploy it using the `oc` CLI client. 
+
+If you choose _I will build and run locally_, unzip and use the ZIP file you downloaded. If you chose _Use {OpenShiftOnline} to build and deploy_ you need to clone the GitHub repository created by {launcher}.
```
Comment (Ladicek) [inline docs/topics/oc-client-deploy-booster-download-note.adoc]: I'd just say "unzip the ZIP file".

**Comment by Ladicek** [comment]
Otherwise LGTM.

**Comment by rhoads-zach** [comment]
thanks @Ladicek, fixed and will merge soon

**Comment by rhoads-zach** [comment]
Merged: https://github.com/openshiftio/appdev-documentation/commit/59298f20e5aeab0948b1a208acfba905ca4ff255

**Pull request closed without merging**